summary_workflows/quantification/Q2_benchmark/Q2_benchmark_specification.md

```diff
@@ -0,0 +1,106 @@
+# Q2 benchmark specification
```

Comment:
I'm not sure why files from `summary_workflows` pop up, but they are already available in the main branch. When merging this branch, the files within `summary_workflows` should be redundant and not cause any problems, hopefully. So I guess we can just leave it as is but be careful when merging.

Reply:
Danger zone